keylime/web_util.py

```diff
@@ -413,7 +413,7 @@ def get_restful_params(urlstring: str) -> Dict[str, Union[str, None]]:
 
     # If first token looks like an API version, validate it and make sure it's supported
     api_version = "0"
-    if path_tokens[0] and len(path_tokens[0]) >= 0 and re.match(r"^v?[0-9]+(\.[0-9]+)?", path_tokens[0]):
+    if path_tokens[0] and len(path_tokens[0]) > 0 and re.match(r"^v?[0-9]+(\.[0-9]+)?", path_tokens[0]):
```

Comment:
Indeed the `if` statement there is super weird. If the first element is `True` imply that the second one is `True` ... always. So why not to drop it?